src/build.cc

```diff
@@ -963,20 +969,18 @@ bool Builder::FinishCommand(CommandRunner::Result* result, string* err) {
     return plan_.EdgeFinished(edge, Plan::kEdgeFailed, err);
   }
 
-  // Restat the edge outputs
-  TimeStamp output_mtime = 0;
-  bool restat = edge->GetBindingBool("restat");
+  TimeStamp most_recent_input = 0;
```

Comment:
Can you rename this variable to `most_recent_input_mtime` so that it isn't easily confused with Edge's `most_recent_input_`?

Reply:
Done